### PR TITLE
feature/MYST-259- myst binary (0.4.0) used in daemon

### PR DESCRIPTION
**Pull request by shroomist**
stuff needed to run 0.4.0 on osx.
could somebody direct me on getting this friendly with Win?

Signed-off-by: Andrej Novikov <jazzaiman@gmail.com>

### CODE REVIEW

File: src/main/dependencies/modules/mysterium-client.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ function bootstrap (container: Container) {
     'mysteriumClient.config',
     ['mysteriumClient.platform', 'mysteriumVpnApplication.config'],
     (platform: string, mysteriumVpnConfig: MysteriumVpnConfig): ClientConfig => {
-      let clientBin = path.join(mysteriumVpnConfig.contentsDirectory, 'bin', 'mysterium_client')
+      let clientBin = path.join(mysteriumVpnConfig.contentsDirectory, 'bin', 'myst')
```
Comment (donce) [inline src/main/dependencies/modules/mysterium-client.js]: !! Value is not updated in `extraFiles`

Could you try building the package to make sure this works?

**Comment by shroomist** [comment]
@donce fixed the `extraFiles` and built locally, it works.
merge, maybe?

File: src/main/dependencies/modules/mysterium-client.js (hunk 1 above)
Comment (Waldz) [reply src/main/dependencies/modules/mysterium-client.js]: > !! Value is not updated in extraFiles

`extraFiles` seems fixed

Comment (Waldz) [reply src/main/dependencies/modules/mysterium-client.js]: >> Could you try building the package to make sure this works?

Testing is not scope of PR review.
Developer creates PR to `master` after testing:
A. Manually
B. CI does it automatically

**Pull request merged**